workers/docker-worker/index.md

```diff
@@ -375,3 +375,47 @@ Example:
   }
 }
 ```
+
+## Coalescing
+
+In some casse, the demand for tasks outstrips the available resources to run those tasks, and pending counts on queues start to grow.
+In many cases, executing some tasks in the queue can render others unnecessary.
+For example, if the test tasks for a later version-control revision succeed, then tests for earlier revisions can be skipped.
+Such earlier tasks are "superseded" by the later task.
+
+Docker-worker supports "coalescing" multiple tasks, executing only one (the "primary task").
+It does so using a "coalescing key", determined as described in the payload schema documentation.
+
+Example:
+
+```js
+{
+    "routes": [
+        "coalesce.foo.v1.build.master"
+    ],
+    "coalescer": {
+        "routePrefix": "coalesce.foo.v1",
+        "url": "https://foo-coalescer.herokuapp.com/"
+    }
+}
+```
+
+This task would have coalescing key `build.master`.
+The worker gets the list of tasks to coalesce by appending the coalescing key to the URL.
+Continuing the example, the foo-coalescer service migh respond with
+
+```js
+{
+    "build.master": [
+        "E5SBRfo-RfOIxh0V4187Qg",
+        "KGt8egfvRaqxczIRgOScaw"
+    ]
```

Comment:
The coalescing key *is* opaque... or are you just repeating the question from above?

The tasks are ordered such that each task can supersede all tasks before it.  The worker executes the highest-indexed task that it successfully claims, which may not be the last one in the list.  So separating it out this way would not work.